S1API/Internal/Patches/QuestPatches.cs

```
@@ -29,62 +30,90 @@
         [HarmonyPatch(typeof(S1Persistence.SaveManager), nameof(S1Persistence.SaveManager.Save), typeof(string))]
         [HarmonyPostfix]
         private static void SaveManager_Save_Postfix(string saveFolderPath)
         {
             try
             {
-                string moddedQuestsPath = Path.Combine(saveFolderPath, "..\\Players\\ModdedQuests");
+                var saveManager = S1Persistence.SaveManager.Instance;
 
-                if (!Directory.Exists(moddedQuestsPath))
-                    Directory.CreateDirectory(moddedQuestsPath);
+                string[] approved = {
+                    "Modded",
+                    Path.Combine("Modded", "Quests")
+                };
 
-                foreach (Quest quest in QuestManager.Quests)
+                foreach (var path in approved)
                 {
+                    if (!saveManager.ApprovedBaseLevelPaths.Contains(path))
+                        saveManager.ApprovedBaseLevelPaths.Add(path);
+                }
 
-                    List<string> dummy = new List<string>();
-
-                    quest.SaveInternal(moddedQuestsPath, ref dummy);
+                // ✅ Create the directory structure
+                string questsPath = Path.Combine(saveFolderPath, "Modded", "Quests");
+                Directory.CreateDirectory(questsPath);
 
+                // ✅ Save only non-vanilla modded quests
+                foreach (Quest quest in QuestManager.Quests)
+                {
+                    if (!quest.GetType().Namespace.StartsWith("ScheduleOne"))
+                    {
+                        List<string> dummy = new List<string>();
+                        quest.SaveInternal(questsPath, ref dummy);
+                    }
                 }
 
             }
             catch (Exception ex)
             {
-                Logger.Error("Failed during SaveManager_Save_Postfix execution.\n" + ex);
-                throw;
```

Why do we not want to throw the exception anymore?